var/spack/repos/builtin/packages/py-nibabel/package.py

```diff
@@ -12,6 +12,7 @@ class PyNibabel(PythonPackage):
     homepage = "https://nipy.org/nibabel"
     pypi     = "nibabel/nibabel-3.2.1.tar.gz"
 
+    version('3.2.2', sha256='b0dcc174b30405ce9e8fec1eab3cbbb20f5c5e4920976c08b22e050b7c124f94')
     version('3.2.1', sha256='4d2ff9426b740011a1c916b54fc25da9348282e727eaa2ea163f42e00f1fc29e')
     version('2.4.1', sha256='f165ff1cb4464902d6594eb2694e2cfb6f8b9fe233b856c976c3cff623ee0e17')
 
```

Comment:
Python 3.6+ is also required in 3.2, not just 3.1

Deps should be type=build/run, not type=run

Reply:
Fixed. Thanks.